version/description

```diff
@@ -0,0 +1,13 @@
+v0.1.0
+
+Anything is better than nothing. When it comes to docker images.
+
+Features:
+* macvtap device plugin, with explicit configuration (#2)
+* macvtap CNI plugin is installed across the cluster (#4)
+
+* add README (#9)
+
+```
+docker pull quay.io/kubevirt/macvtap-cni
```

Comment:
This needs to be pinned to a version. Otherwise it does not make much sense to keep it in release notes. But honestly, I don't think it make sense to keep it at all. You can really use the image alone.

Reply:
Right.